client/main.lua

```diff
@@ -56,7 +56,7 @@ for i = 1, #config.locations do
             debug = config.polyDebug,
             inside = function()
                 if not lib.progressActive() then
-                    qbx.drawText3d({text = locale('wash_prompt', price), coords = coords})
+                    qbx.drawText3d({text = locale('wash_prompt_dt', price), coords = coords})
```

Comment:
Why the rename?

Reply:
Because the wash_prompt doesn't have the key you're supposed to press to wash your car, since the other one has a "dt" at the end that I think stands for drawtext, I believe the first one was intented to be used for target and second one for drawtext